core/src/test/scala/org/apache/spark/sql/delta/DeltaRestartSessionSuite.scala

```diff
@@ -18,19 +18,24 @@ package org.apache.spark.sql.delta
 
 import org.apache.spark.SparkFunSuite
 import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.delta.sources.DeltaSQLConf
 
 class DeltaRestartSessionSuite extends SparkFunSuite {
 
   test("restart Spark session should work") {
     withTempDir { dir =>
-      var spark = SparkSession.builder().master("local[2]").getOrCreate()
+      var spark = SparkSession.builder().master("local[2]")
+        .config(DeltaSQLConf.DELTA_REQUIRED_SPARK_CONFS_CHECK.key, "false")
+        .getOrCreate()
       try {
         val path = dir.getCanonicalPath
         spark.range(10).write.format("delta").mode("overwrite").save(path)
         spark.read.format("delta").load(path).count()
 
         spark.stop()
-        spark = SparkSession.builder().master("local[2]").getOrCreate()
+        spark = SparkSession.builder().master("local[2]")
+          .config(DeltaSQLConf.DELTA_REQUIRED_SPARK_CONFS_CHECK.key, "false")
```

Comment:
Could you set the catalog rather than disabling the check?

Reply:
done